cflib/crazyflie/mem.py

```diff
@@ -194,6 +197,68 @@ def disconnect(self):
         self._write_finished_cb = None
 
 
+class LEDMemoryDriverMemory(MemoryElement):
```

Comment:
What is the difference between this one and the class LEDMemoryDriverMemory ? The explanation says exactly the same function. Why is the already existing one not sufficient for this or can be adjusted to do what you want it to do?

Reply:
LEDDriverMemory is handling the memory to show a fixed static pattern across the leds of the deck,
while LEDMemoryDriverMemory is handling the memory to show a changing pattern across the leds of the deck. So possibility to upload a show of multiple led patterns and from the moment you enable the lights, it will go through that show pattern by patterns with the ability to specify if the next pattern should fade in and what leds that should be enabled.

Reply:
Ah yeah oke I understand. But currently the explanation of this class is a direct copy:
    _"""Memory interface for using the LED-ring mapped memory for setting RGB
       values for all the LEDs in the ring"""_
This should be adjusted to explain the difference.

Reply:
In general, it might be better to have a different name for it. LEDSequenceDriverMemory perhaps? Or just merge it with the LEDdrivermemory. We kind of want to avoid duplicates if you can understand

Reply:
Is something like LEDTimingDriverMemory good? As the memory layouts are totally different, I think it is best not to merg with LEDDriverMemory

Reply:
Yeah something like that :)